src/util/system.cpp

```diff
@@ -723,22 +736,7 @@ fs::path GetDefaultDataDir()
 #endif
 }
 
-namespace {
-fs::path StripRedundantLastElementsOfPath(const fs::path& path)
-{
-    auto result = path;
-    while (result.filename().string() == ".") {
-        result = result.parent_path();
-    }
-
-    assert(fs::equivalent(result, path));
-    return result;
-}
-} // namespace
-
 static fs::path g_blocks_path_cache_net_specific;
-static fs::path pathCached;
-static fs::path pathCachedNetSpecific;
```

Comment:
In commit "Move StripRedundantLastElementsOfPath before ArgsManager class." (c44993ad96d649be48c94d4ae38d354d609a7282)

This commit seems not to compile because these variables are removed

```
  CXX      libbitcoin_server_a-net.o
util/system.cpp:771:37: error: use of undeclared identifier 'pathCachedNetSpecific'
    fs::path &path = fNetSpecific ? pathCachedNetSpecific : pathCached;
                                    ^
util/system.cpp:771:61: error: use of undeclared identifier 'pathCached'
    fs::path &path = fNetSpecific ? pathCachedNetSpecific : pathCached;
                                                            ^
util/system.cpp:809:5: error: use of undeclared identifier 'pathCached'; did you mean 'csPathCached'?
    pathCached = fs::path();
    ^~~~~~~~~~
    csPathCached
util/system.cpp:740:23: note: 'csPathCached' declared here
static RecursiveMutex csPathCached;
                      ^
util/system.cpp:809:16: error: no viable overloaded '='
    pathCached = fs::path();
    ~~~~~~~~~~ ^ ~~~~~~~~~~
./sync.h:89:16: note: candidate function (the implicit copy assignment operator) not viable: no known conversion from 'fs::path' to 'const AnnotatedMixin<std::recursive_mutex>' for 1st argument
class LOCKABLE AnnotatedMixin : public PARENT
               ^
util/system.cpp:810:5: error: use of undeclared identifier 'pathCachedNetSpecific'
    pathCachedNetSpecific = fs::path();
    ^
```

Reply:
Fixed, thanks.